src/main/archetype/all/pom.xml

```diff
@@ -186,6 +208,19 @@
             <type>zip</type>
         </dependency>
 
+#if ( $isSpaProject )
+        <!-- SPA Project Core (includes hierarchy page) -->
```

Comment:
while not directly related to this PR, why was the Hierarchy Page Impl not added to AEM Core Components? Is spa.project.core.call open source?

Reply:
The hierarchical extension of the `Page` component is not part of Core Components because it's very SPA-specific. The library is currently closed source.